make_prg/seq_utils.py

```diff
@@ -54,14 +54,23 @@ def ungap(seq: str) -> str:
     return seq.replace("-", "")
 
 
-def get_interval_seqs(interval_alignment: MSA):
+def count(iterable) -> int:
```

Comment:
is this not just the `len` function?

Reply:
Good spot. `len` only applies to containers, e.g. a list, not generators, like the `remove_duplicates` function.
`count` is used on generator functions: e.g. prg_builder.py: L130-132. The advantage of using generator functions is they can be chained in a functional programming way, and this avoids building lists!